### ISSUE TITLE
parallel pistons won't extend

### ISSUE THREAD

**Issue by AlienXtream** [tsunamayo/Starship-EVO#2637]
### Short Description of the issue:

having multiple pistons activate in parallel (as shown in image) prevents extension of piston

### Steps to reproduce the issue:

place multiple pistons in parallel and have a child grid span across them all
expected result is that all 3 pistons (in this example) should extend.
additionally having the child grid be truly attached to all 3 so that the child grid remains in tact so long as at least one of its attached pistons remains
### Environment
- Starship EVO version: 20w28a
- Graphics Card: N/A

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_
![image](https://user-images.githubusercontent.com/12606809/86576205-c26e0080-bfbb-11ea-8b5a-ce3495cb6b9a.png)

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by Kaiser-Indrasil** [comment]
You have to set the child-less pistons extend slower compared to the weight-lifting one since the rule of thumb is, if piston's path is obstructed, it just stops extending.

**Comment by AlienXtream** [comment]
its still a bug/unexpected behaviour and should be addressed to be more intuitive

**Comment by tsunamayo** [comment]
Yes I can understand this, but it is quite hard to do behind, maybe one day. Cheers

**Comment by AlienXtream** [comment]
as a temp "fix" might i suggest having pistons with mass on them move fractionally faster (like, 1.001x faster or something) than a lone piston so at least the mechanism extends and LOOKS as intended. not ideal i know but as mentioned, it would be a temp fix until some form of grid merging is properly introduced and implemented
